lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -74,9 +79,9 @@ namespace Js
 #endif
         if (VarIs<JavascriptProxy>(target))
         {
-            if (VarTo<JavascriptProxy>(target)->target == nullptr)
+            if (VarTo<JavascriptProxy>(target)->IsRevoked())
             {
-                JavascriptError::ThrowTypeError(scriptContext, JSERR_InvalidProxyArgument, _u("target"));
+                JavascriptError::ThrowTypeError(scriptContext, JSERR_ErrorOnRevokedProxy, _u("target"));
```

Comment:
This change and the similar one on line 97 are causing the test fails. As there's a test that compares the text of the error message in this case - if keeping this error message update you need to update test/es6/proxytest9.baseline

Reply:
I think this should be reverted. 
Now the error message is:
```
method xxx is called on a revoked Proxy object
```
However, what is causing error should be arguments.